plugin/src/main/java/at/helpch/chatchat/util/FormatUtils.java

```diff
@@ -40,25 +38,37 @@ private FormatUtils() {
         return format.parts().stream()
                 .map(part -> PlaceholderAPI.setPlaceholders(player, part))
                 .map(part -> part.replace("%message%", message))
-                .map(part -> replaceRecipientPlaceholder(player, part))
                 .map(FormatUtils::parseToMiniMessage)
                 .collect(Component.toComponent());
     }
 
+    public static @NotNull Component parseFormat(
+        @NotNull final Format format,
+        @NotNull final Player player,
+        @NotNull final Player receiver,
+        @NotNull final String message) {
+        return format.parts().stream()
+            .map(part -> PlaceholderAPI.setPlaceholders(player, part))
+            .map(part -> part.replace("%message%", message))
+            .map(part -> replaceRecipientPlaceholder(receiver, part))
+            .map(FormatUtils::parseToMiniMessage)
+            .collect(Component.toComponent());
```

Comment:
Could this be `Component::toComponent`?

Reply:
Possibly. Not sure. its what was used for the other method. just copy pasted it :))

Reply:
Try it out and see! Then you can change the other one if it works 😌

Reply:
Component is not a functional interface, no

Reply:
Kaliber is right. 
![image](https://user-images.githubusercontent.com/52609756/158032904-db69ecc3-f72b-4709-b504-d8cd750aab8a.png)